src/Surfnet/AzureMfa/Domain/Exception/InvalidMfaNameIdException.php

```diff
@@ -20,6 +20,6 @@
 
 use Exception;
 
-class InvalidMFANameIdException extends Exception
+class InvalidMfaNameIdException extends Exception
```

Comment:
What is the reason why this is changed?

Reply:
MFA was capitalized incorrectly according to the camel case rules